web/app/package.json

```diff
@@ -37,8 +35,11 @@
     "babel-eslint": "^8.0.3",
     "babel-loader": "^7.1.2",
     "babel-plugin-import": "^1.5.0",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
```

Comment:
Should we remove the "babel-preset-react" dev dependency now that we're no longer using the "react" preset in .babelrc?

Reply:
I can strip these down quite a bit more actually ....